assets/index.pug

```diff
@@ -7,6 +7,9 @@ html
     meta(name="viewport", content="width=device-width, initial-scale=1")
     meta(name="theme-color", content="#0C7CD5")
     script(src="/socket.io/socket.io.js")
-
+    style(type='text/css').
+      esn-search-advanced-form{
+          display:none;
+      }
```

Comment:
```suggestion
    style(type='text/css').
      esn-search-header {
          display: none;
      }
```

I don't know if this is a clean way to do it though.

Reply:
There's another issue related to this which will bring back the search bar, so I think for now this is probably ok

Reply:
Can you please link that issue in the description + a bit of explanation so that in the future we have something for reference

Reply:
Fixed and update the description